db/migrate/20240418180337_add_error_code_to_workflow_execution.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Migration to add error_code column to workflow_executions table
+class AddErrorCodeToWorkflowExecution < ActiveRecord::Migration[7.1]
+  def change
+    add_column :workflow_executions, :error_code, :integer
```

Comment:
**nitpick (blocking):** can we change this to `http_error_code` since we may want to add an `error_code` in the future that is an enum like `CommunicationError` etc that is not entirely related to `http_error_code`.

Reply:
Changed in https://github.com/phac-nml/irida-next/pull/546/commits/ebfe278e975461c66edf27a7560aad0a453916eb